compiler-rt/test/fuzzer/uncaught-exception.test

```diff
@@ -6,8 +6,6 @@ RUN: %cpp_compiler %S/UncaughtException.cpp -o %t-UncaughtException
 
 # Clang will fail the test with 'deadly signal', but other compilers may fail with different error messages.
 # For example, msvc fails with 'uncaught C++ exception'. So the error we check depends on the compiler target.
-RUN: not %run %t-UncaughtException 2>&1 | FileCheck %s --check-prefixes=CHECK-CRASH,%if target={{.*-windows-msvc.*}} %{CHECK-MSVC%} %else %{CHECK-ERROR%}
+RUN: not %run %t-UncaughtException 2>&1 | FileCheck %s
 
-CHECK-ERROR: ERROR: libFuzzer: deadly signal
-CHECK-MSVC: ERROR: libFuzzer: uncaught C++ exception
-CHECK-CRASH: Test unit written to ./crash
+CHECK: {{==.*== ERROR: libFuzzer: deadly signal|==.*== ERROR: libFuzzer: uncaught C++ exception}} 
```

Comment:
This probably works, though it's not necessary to match on `==.*== ERROR`.

It could also be refactored to 
```
CHECK: ERROR: libFuzzer: {{deadly signal|uncaught C++ exception}}
```

Reply:
Make sense. Done